software/control/microcontroller.py

```diff
@@ -568,20 +577,26 @@ def configure_actuators(self):
         self.wait_till_operation_is_completed()
         self.set_leadscrew_pitch(AXIS.Z,SCREW_PITCH_Z_MM)
         self.wait_till_operation_is_completed()
+        self.set_leadscrew_pitch(AXIS.W,SCREW_PITCH_W_MM)
+        self.wait_till_operation_is_completed()
         # stepper driver (microstepping,rms current and I_hold)
         self.configure_motor_driver(AXIS.X,MICROSTEPPING_DEFAULT_X,X_MOTOR_RMS_CURRENT_mA,X_MOTOR_I_HOLD)
         self.wait_till_operation_is_completed()
         self.configure_motor_driver(AXIS.Y,MICROSTEPPING_DEFAULT_Y,Y_MOTOR_RMS_CURRENT_mA,Y_MOTOR_I_HOLD)
         self.wait_till_operation_is_completed()
         self.configure_motor_driver(AXIS.Z,MICROSTEPPING_DEFAULT_Z,Z_MOTOR_RMS_CURRENT_mA,Z_MOTOR_I_HOLD)
         self.wait_till_operation_is_completed()
+        self.configure_motor_driver(AXIS.W,MICROSTEPPING_DEFAULT_W,W_MOTOR_RMS_CURRENT_mA,W_MOTOR_I_HOLD)
+        self.wait_till_operation_is_completed()
```

Comment:
@veerwang this should only happen when the microscope has Squid filter wheel

Reply:
OK, I would revise it.